packages/falcon-magento2-api/src/index.js

```diff
@@ -897,13 +900,23 @@ module.exports = class Magento2Api extends Magento2ApiBase {
    * @returns {CountryList} parsed country list
    */
   async countryList() {
-    const response = await this.getAuth('/directory/countries', {}, { context: { isAuthRequired: false } });
-    const countries = response.map(item => ({
-      code: item.id,
-      englishName: item.full_name_english,
-      localName: item.full_name_locale,
-      regions: item.available_regions || []
-    }));
+    const countries = await this.getAuth(
+      '/directory/countries',
+      {},
+      {
```

Comment:
```suggestion
      {
        cacheOptions: { ttl: 86400 }, // 24 hours cache
```

I think it would be safe to cache that for 24 hours (doing this on API class, because other methods are also using this getter)

Reply:
Do you know what is the default TTL?

Reply:
@nimobeeren there's no default TTL (it works automatically if a backend sends `max-age` response header - this way TTL is set from that value). Otherwise - there's no default value.
So sometimes, we need to force this cache TTL. _It's just a suggestion_

Reply:
So when a backend doesn't send such a header, is the value cached indefinitely or not at all? I just want to know how it works :)

Reply:
@nimobeeren it works for positive values only, so if there's no `max-age` header - nothing will be cached (meaning - refetch from the source each time).